src/devices/tuya.ts

```diff
@@ -5310,6 +5310,165 @@ const definitions: Definition[] = [
+            new exposes.Enum('light_mode_l8', ea.STATE_SET, ['normal', 'on', 'off', 'flash']).
+                withDescription(`'Sets the indicator mode of l8.
+                    Normal: Orange while off and white while on.
+                    On: Always white. Off: Always orange.
+                    Flash: Flashes white when triggered.
+                    Note: Orange light will turn off after light off delay, white light always stays on'`),
+        ],
+        endpoint: (device) => {
+            return {'l1': 1, 'l2': 1, 'l3': 1, 'l4': 1, 'l5': 1, 'l6': 1, 'state': 1, 'backlight': 1, 'l7': 1, 'l8': 1};
+        },
+        meta: {
+            multiEndpoint: true,
+            tuyaDatapoints: [
+                [1, 'state_l1', tuya.valueConverter.onOff],
+                [2, 'state_l2', tuya.valueConverter.onOff],
+                [3, 'state_l3', tuya.valueConverter.onOff],
+                [4, 'state_l4', tuya.valueConverter.onOff],
+                [5, 'state_l5', tuya.valueConverter.onOff],
+                [6, 'state_l6', tuya.valueConverter.onOff],
+                [16, 'backlight_mode', tuya.valueConverter.onOff],
+                [29, 'power_on_behavior_l1', tuya.valueConverterBasic.lookup({'off': tuya.enum(0), 'on': tuya.enum(1), 'memory': tuya.enum(2)})],
+                [30, 'power_on_behavior_l2', tuya.valueConverterBasic.lookup({'off': tuya.enum(0), 'on': tuya.enum(1), 'memory': tuya.enum(2)})],
+                [31, 'power_on_behavior_l3', tuya.valueConverterBasic.lookup({'off': tuya.enum(0), 'on': tuya.enum(1), 'memory': tuya.enum(2)})],
+                [32, 'power_on_behavior_l4', tuya.valueConverterBasic.lookup({'off': tuya.enum(0), 'on': tuya.enum(1), 'memory': tuya.enum(2)})],
+                [33, 'power_on_behavior_l5', tuya.valueConverterBasic.lookup({'off': tuya.enum(0), 'on': tuya.enum(1), 'memory': tuya.enum(2)})],
+                [34, 'power_on_behavior_l6', tuya.valueConverterBasic.lookup({'off': tuya.enum(0), 'on': tuya.enum(1), 'memory': tuya.enum(2)})],
+                [101, 'action', tuya.valueConverter.static('scene_1')],
+                [102, 'action', tuya.valueConverter.static('scene_2')],
+                [103, 'action', tuya.valueConverter.static('scene_3')],
+                [104, 'action', tuya.valueConverter.static('scene_4')],
+                [105, 'presence', tuya.valueConverter.raw],
+                [106, 'delay', tuya.valueConverter.raw],
+                [108, 'action', tuya.valueConverter.static('scene_5')],
+                [109, 'action', tuya.valueConverter.static('scene_6')],
+                [110, 'action', tuya.valueConverter.static('scene_7')],
+                [111, 'action', tuya.valueConverter.static('scene_8')],
+                [112, 'state_l7', tuya.valueConverter.onOff],
+                [113, 'state_l8', tuya.valueConverter.onOff],
+                [114, 'mode_l1', tuya.valueConverterBasic.lookup({'switch': tuya.enum(0), 'scene': tuya.enum(1)})],
```

Comment:
For both `mode` and `light_mode` add a new valueConverter (https://github.com/Koenkk/zigbee-herdsman-converters/blob/3440313f70db9c5b4c33646091dbf704bd5db985/src/lib/tuya.ts#L374)